lib/src/components/NxTile/__tests__/NxTile.test.tsx

```diff
@@ -4,68 +4,92 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
+import { ComponentType } from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxTile from '../NxTile';
 
+function testMergedClassNames(Component: ComponentType<{}>) {
+  return () => {
+    const renderEl = rtlRenderElement(Component, {});
+
+    const defaultEl = renderEl()!,
+        customEl = renderEl({ className: 'foo' });
+
+    expect(customEl).toHaveClass('foo');
+
+    for (const cls of Array.from(defaultEl.classList)) {
+      expect(customEl).toHaveClass(cls);
+    }
+  };
+}
+
+function quickRender(Component: ComponentType<{}>) {
```

Comment:
I think we can just use render from RTL directly and remove this function

Reply:
> I think we can just use render from RTL directly and remove this function

Yep, that's cleaner, updated with render here: [76d7a29](https://github.com/sonatype/sonatype-react-shared-components/pull/959/commits/76d7a2945d216d072ec76ec467e06c731a24a0d0)